### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by weslenng**

### CODE REVIEW

File: src/Crunchyroll.re [hunk 1]
```diff
@@ -0,0 +1,112 @@
+open Cheerio;
+open Config;
+open Util;
+
+type options = {
```
Comment (EduardoRFS) [inline src/Crunchyroll.re:5]: Also when making FFI, you should make a wrapper to it, to fix the currying and avoid being too much implementation dependent

File: src/Index.re [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  let%Async anime = popular->get_anime;
+  let%Async episode = anime->get_episode;
+  let%Async file_path = episode->from_crunchyroll(folder);
+
+  file_path->split_file(folder);
+};
+
+let prepare_music = folder => {
+  ()->get_music->from_youtube(folder);
```
Comment (EduardoRFS) [inline src/Index.re:21]:
```suggestion
  get_music()->from_youtube(folder);
```

File: src/Index.re [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+};
+
+[@bs.module "path"] [@bs.variadic]
+external join: array(string) => string = "join";
```
Comment (EduardoRFS) [inline src/Index.re:25]: This is another reason to not expose FFI, you don't want `array(string)` you want `list(string)`

File: src/Util.re [hunk 4]
```diff
@@ -0,0 +1,33 @@
+let get_random = (maximum, minimum) => {
+  let inclusive = 1;
+
+  Js_math.(
+    (()->random *. (maximum - minimum + inclusive)->float_of_int)->floor_int
+    + minimum
+  );
+};
```
Comment (EduardoRFS) [inline src/Util.re:8]: This API is quite weird, who does max then min?

**Comment by weslenng** [comment]
@EduardoRFS Thanks <3

**Pull request closed without merging**